### ISSUE TITLE
Rename this repo to web-features

### ISSUE THREAD

**Issue by foolip** [web-platform-dx/feature-set#252]
I was a bit devious when I published https://www.npmjs.com/package/web-features, picking a different name than the repo. I felt `web-features` would make more sense alongside https://www.npmjs.com/package/web-specs, which will often be used together with this repo.

This has created a problem, namely that I keep having to say "feature-set slash web-features" to refer to this project.

Let's rename it!

@ddbeck WDYT?

**Comment by ddbeck** [comment]
@foolip Yes, I agree with this. Naming things is hard enough as it is, so there's no need to have _two_ of them.

Stuff that would need to happen to fix this:

- [ ] Rename the repo
- [ ] Find and check each instance of "feature-set" / "feature set" in the repo's Markdown files
- [ ] Announce this change in some useful places (i.e., the WebDX mailing list and meeting)

Am I missing anything?

**Comment by foolip** [comment]
That sounds right, I'm not aware of other places using the repo name. But maybe use GitHub search to see if there are cases we haven't been involved with ourselves.

**Comment by tidoust** [comment]
Suggestion to rename the repo was [discussed in last week's WebDX CG meeting](https://docs.google.com/document/d/1ree75ImLZjf60lTZ3BhCaLHygxgywr7SBXp-q0xPs8A/edit#heading=h.r58pqm3rbjbt) with general support for the change. Feel free to go ahead with this.

**Comment by ddbeck** [comment]
Draft announcement to send to the mailing list, once #335 is approved:

> Hi all,
> 
> [As discussed in July](https://github.com/web-platform-dx/feature-set/issues/252), to reduce some confusion around the names of the feature set repo and package, we're renaming `web-platform-dx/feature-set` to `web-platform-dx/web-features` on \[day]. This means that the repo will match the published package (`web-features`) on npm.
> 
> Let me repeat myself: starting on \[day], `feature-set` will be `web-features`.
> 
> This ought to be minimally annoying, as GitHub will redirect existing links to the repository and I've prepared PRs to point to the new name and URL within the web-platform-dx org on GitHub and on MDN. That said, if you run into issues, please @ me (ddbeck) on issues or PRs for this — I'll endeavor to fix them promptly.
> 
> Thank you!

**Comment by ddbeck** [comment]
I sent the email this morning. I will now start renaming the repo, applying #335, and opening one a PR for mdn/content.

**Comment by ddbeck** [comment]
https://github.com/mdn/content/pull/28924 has been approved and I expect it to merge shortly. Closing this up. Thanks, everyone!
